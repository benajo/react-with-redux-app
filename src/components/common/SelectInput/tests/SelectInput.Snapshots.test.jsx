import React from 'react';
import renderer from 'react-test-renderer';
import SelectInput from '../SelectInput';

it('render SelectInput component', () => {
  const tree = renderer.create(
    <SelectInput
      name="list"
      label="List"
      onChange={jest.fn()}
    />,
  );
  expect(tree).toMatchSnapshot();
});
