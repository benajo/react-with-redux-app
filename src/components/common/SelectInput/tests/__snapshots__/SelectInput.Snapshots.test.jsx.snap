// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render SelectInput component 1`] = `
<div
  className="form-group"
>
  <label
    htmlFor="list"
  >
    List
  </label>
  <div
    className="field"
  >
    <select
      className="form-control"
      name="list"
      onChange={[MockFunction]}
      value=""
    >
      <option
        value=""
      >
        
      </option>
    </select>
    
  </div>
</div>
`;
